java/arcs/core/storage/Handle.kt

```diff
@@ -45,6 +45,22 @@ interface Callbacks<Data : CrdtData, Op : CrdtOperationAtTime, T> {
     fun onDesync(handle: Handle<Data, Op, T>) = Unit
 }
 
+/**
+ * Basic wrapper around [Callbacks] which lets you supply lambdas for each method instead of
+ * implementing the whole interface yourself.
+ */
+class LambdaCallbacks<Data : CrdtData, Op : CrdtOperationAtTime, T>(
+    private val onSync: () -> Unit,
```

Comment:
You've changed the contract here by not supplying the handle, this could make it less useful in circumstances where someone might use a single lambda to listen to multiple handles.  I think Piotr/Sarah should review this also because this changes an API in the particle SDK.